zcash_client_sqlite/src/lib.rs

```diff
@@ -196,6 +204,19 @@ impl<C: Borrow<rusqlite::Connection>, P: consensus::Parameters> TransparentInput
 {
     type Error = SqliteClientError;
 
+    fn get_unspent_transparent_output(
+        &self,
+        _outpoint: &OutPoint,
+    ) -> Result<Option<WalletTransparentOutput>, Self::Error> {
+        #[cfg(feature = "transparent-inputs")]
+        return wallet::get_unspent_transparent_output(self.conn.borrow(), _outpoint);
+
+        #[cfg(not(feature = "transparent-inputs"))]
+        panic!(
+            "The wallet must be compiled with the transparent-inputs feature to use this method."
+        );
+    }
```

Comment:
This makes me think that the removal of the feature flag on the trait definition in `zcash_client_backend` is a rebase bug. Please fix if so.

In any case, this impl should be adjusted (because the entire trait impl is guarded by the feature flag):
```suggestion
    fn get_unspent_transparent_output(
        &self,
        outpoint: &OutPoint,
    ) -> Result<Option<WalletTransparentOutput>, Self::Error> {
        wallet::get_unspent_transparent_output(self.conn.borrow(), outpoint)
    }
```

Reply:
The removal of the feature flag is not a bug; the alternative here to a panic is to return `Ok(None)` but I think that this would obscure the fact that the user is doing something wrong with the way that they're using the library.

Reply:
Removing the feature flag on the trait definition seems fine (per your rationale in the other comment). But this PR keeps the feature flag on the whole trait implementation, meaning that my suggestion above is entirely correct in that context.